### PR TITLE
Rcint 6558

### PR DESCRIPTION
**Pull request by huanhulan**

### CODE REVIEW

**Comment by RingCentralIntegrationDev** [comment]
Build successful! You can check your changes here: https://ec2-52-17-190-173.eu-west-1.compute.amazonaws.com/job/rc-js-widget-pr/69/

**Comment by RingCentralIntegrationDev** [comment]
Build successful! You can check your changes here: https://ec2-52-17-190-173.eu-west-1.compute.amazonaws.com/job/rc-js-widget-pr/70/

File: .idea/ringcentral-js-widgets.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
```
Comment (embbnux) [inline .idea/ringcentral-js-widgets.iml]: remove unused config file

**Comment by RingCentralIntegrationDev** [comment]
Build successful! You can check your changes here: https://ec2-52-17-190-173.eu-west-1.compute.amazonaws.com/job/rc-js-widget-pr/72/

**Comment by RingCentralIntegrationDev** [comment]
Build successful! You can check your changes here: https://ec2-52-17-190-173.eu-west-1.compute.amazonaws.com/job/rc-js-widget-pr/75/

**Pull request merged**